packages/core/directives/polyline.ts

```diff
@@ -42,6 +42,11 @@ export class AgmPolyline implements OnDestroy, OnChanges, AfterContentInit {
    */
   @Input() clickable: boolean = true;
 
+  /**
+   * icons of the polyline
+   */
+  @Input() icons: IconSequence[] = undefined;
```

Comment:
can you please add this type to this file?: https://github.com/SebastianM/angular-google-maps/blob/master/packages/core/map-types.ts

Reply:
I'm missing some logic for update scenarios of this input.

Reply:
`IconSequence` added to `map-types.ts`

Reply:
> I'm missing some logic for update scenarios of this input.

Is this an action for me to take? If so can you expand on the changes you are looking for?

Reply:
Yeah, I can do it if you have no time for it. With the current PR, the user cannot update the icons at runtime.